src/FffGcodeWriter.cpp

```diff
@@ -1201,8 +1201,8 @@ void FffGcodeWriter::addMeshLayerToGCode(const SliceDataStorage& storage, const
     {
         addMeshOpenPolyLinesToGCode(mesh, mesh_config, gcode_layer);
     }
-
-    gcode_layer.setMesh(0);
+    //remove this as wipe movements would get put into this category
+    //gcode_layer.setMesh("NONMESH");
```

Comment:
Removing this causes the next stuff to be marked as belonging to this mesh. The next stuff could be a prime tower or brim or basically anything.

Reply:
I believe these were the extra movements following the skin to minimize oozing. Its not a huge deal, but it does require more travel moves if an object is cancelled. I'll fix this, but perhaps there is a way for these moves to be included in the mesh block.

Reply:
Ah, probably coasting then.

I don't think this is a big deal, but you're right in that it should belong to the mesh.